helm/polaris/values.yaml

```diff
@@ -0,0 +1,374 @@
+  initialDelaySeconds: 5
+  # -- How often (in seconds) to perform the probe. Minimum value is 1.
+  periodSeconds: 10
+  # -- Minimum consecutive successes for the probe to be considered successful after having failed. Minimum value is 1.
+  successThreshold: 1
+  # -- Minimum consecutive failures for the probe to be considered failed after having succeeded. Minimum value is 1.
+  failureThreshold: 3
+  # -- Number of seconds after which the probe times out. Minimum value is 1.
+  timeoutSeconds: 10
+
+# -- Advanced configuration via Environment Variables.
+# Extra environment variables to add to the Polaris server container.
+# You can pass here any valid EnvVar object:
+# https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#envvar-v1-core
+# This can be useful to get configuration values from Kubernetes secrets or config maps.
+extraEnv:
+  []
+#  - name: AWS_STORAGE_BUCKET
+#    value: s3://xxxxx/
+#  - name: AWS_ACCESS_KEY_ID
+#    valueFrom:
+#      secretKeyRef:
+#        name: aws-secret
+#        key: access_key_id
+#  - name: AWS_SECRET_ACCESS_KEY
+#    valueFrom:
+#      secretKeyRef:
+#        name: aws-secret
+#        key: secret_access_key
+
+# -- Configures for persistence.xml (ensure the key name is 'persistence.xml')
+persistenceConfigSecret: polaris-secret
+
+# -- Configures for polaris-server.yml
+polarisServerConfig:
+  server:
+    # Maximum number of threads.
+    maxThreads: 200
+
+    # Minimum number of thread to keep alive.
+    minThreads: 10
+    applicationConnectors:
+      # HTTP-specific options.
+      - type: http
+
+        # The port on which the HTTP server listens for service requests.
+        port: 8181
+
+    adminConnectors:
+      - type: http
+        port: 8182
+
+    # The hostname of the interface to which the HTTP server socket wil be found. If omitted, the
+    # socket will listen on all interfaces.
+    #bindHost: localhost
+
+    #  ssl:
+    #    keyStore: ./example.keystore
+    #    keyStorePassword: example
+    #
+    #    keyStoreType: JKS # (optional, JKS is default)
+
+    # HTTP request log settings
+    requestLog:
+      appenders:
+        # Settings for logging to stdout.
+        - type: console
+
+        # Settings for logging to a file.
+        - type: file
+
+          # The file to which statements will be logged.
+          currentLogFilename: ./logs/request.log
+
+          #  When the log file rolls over, the file will be archived to requests-2012-03-15.log.gz,
+          # requests.log will be truncated, and new statements written to it.
+          archivedLogFilenamePattern: ./logs/requests-%d.log.gz
+
+          # The maximum number of log files to archive.
+          archivedFileCount: 14
+
+          # Enable archiving if the request log entries go to the their own file
+          archive: true
+
+  # Either 'jdbc' or 'polaris'; specifies the underlying delegate catalog
+  baseCatalogType: "polaris"
```

Comment:
question: I have to comment out this line `# baseCatalogType: "polaris"` in order to deploy successfully to my cluster. What is this for?

Reply:
I think that is obsolete and got removed from main as part of https://github.com/polaris-catalog/polaris/pull/143/files. I had updated the PR with latest `polaris-server.yml`. Then to answer your question, for any config under `.polarisServerConfig`, you can make change to it as how you will deal with a regular `polaris-server.yml`. As the config changes all the time and I don't want to hard coded the logic in config validation, I am basically reading this section and convert to yaml for configmap. So if some other settings are added/removed later, all you needed to add/remove them accordingly.

Reply:
Thank you for your explanation and thanks for the helm chart. It's super useful for me to deploy to my cluster :)

Reply:
Anytime.

Reply:
@dimas-b see above for the issue you reported.